newswires/client/src/SearchMenu.tsx

```diff
@@ -0,0 +1,34 @@
+import { EuiButtonIcon, EuiPopover } from '@elastic/eui';
+import { useState } from 'react';
+import { SearchBox } from './SearchBox.tsx';
+
+export const SearchMenu = () => {
```

Comment:
as mentioned in other comment, I think we might still want to use this for `xs` breakpoints, but if we don't then it looks like this component is unused atm?